scss/_list-group.scss

```diff
@@ -173,6 +173,27 @@
   }
 }
 
+// Horizontal flush list items
+//
+// Remove borders and border-radius to keep horizontal list group items edge-to-edge. Most
+// useful within other components (e.g., cards).
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+
+    .list-group-flush.list-group-horizontal#{$infix} {
+      @include border-radius(0);
```

Comment:
I think we can safely remove this line since it already comes with `.list-group-flush`

Reply:
Yeah, right.
Done.